luigi_pipeline/seqr_loading.py

```diff
@@ -138,12 +138,22 @@ def read_input_write_mt(self):
 
     def annotate_old_and_split_multi_hts(self, mt):
         """
-        Saves the old allele and locus because while split_multi does this, split_multi_hts drops this. Will see if
-        we can add this to split_multi_hts and then this will be deprecated.
+        Saves the old allele and locus because while split_multi saves the fields
+        split_multi_hts drops. Will see if we can add this to split_multi_hts and
+        then this will be deprecated.
+
+        Additional logic is added here to support VCFs which contain biallelic and
+        multiallelic rows.  The `split_multi_hts` function, by default, will fail if there are both 
+        split and unsplit loci.  We want to only run the split on the multiallelic rows
+        for performance reasons, rather than allowing a shuffle to happen.
         :return: mt that has pre-annotations
         """
         # Named `locus_old` instead of `old_locus` because split_multi_hts drops `old_locus`.
-        return hl.split_multi_hts(mt.annotate_rows(locus_old=mt.locus, alleles_old=mt.alleles))
+        bi = mt.filter_rows(hl.len(mt.alleles) == 2)
+        bi = bi.annotate_rows(was_split=False, locus_old=mt.locus, alleles_old=mt.alleles)
+        multi = mt.filter_rows(hl.len(mt.alleles) > 2)
+        split = hl.split_multi_hts(multi.annotate_rows(locus_old=mt.locus, alleles_old=mt.alleles))
```

Comment:
I think this should be
```suggestion
        split = hl.split_multi_hts(multi.annotate_rows(locus_old=multi.locus, alleles_old=multi.alleles))
```